config/routes.rb

```diff
@@ -14,6 +14,8 @@
     root 'home#index'
   end
 
+  get :cieszyn, to: redirect('pl/cieszyn/welcome')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.